### ISSUE TITLE
UndefVarError: σ not defined

### ISSUE THREAD

**Issue by lagefreitas** [NaelsonDouglas/DistributedMachineLearningThesis#8]
It complains about `\sigma` (σ) var not defined.

Please remark that owing to Issue #7 I had to call by hand Julia 0.6.4:

```bash
root@8b2c10453516:/DistributedMachineLearningThesis/src# ~/julia/bin/julia -L master_summary.jl -e "execute_experiment()" 4 100 f1 1234 4 2

[...]

./event.jl:73
while loading no file, in expression starting on line 0
ERROR: On worker 2:
UndefVarError: σ not defined
macro expansion at /root/.julia/v0.6/Distributions/src/utils.jl:5 [inlined]
Type at /root/.julia/v0.6/Distributions/src/univariate/continuous/normal.jl:28 [inlined]
Type at /root/.julia/v0.6/Distributions/src/univariate/continuous/normal.jl:32 [inlined]
Type at /root/.julia/v0.6/Distributions/src/univariate/continuous/normal.jl:33 [inlined]
f1 at /DistributedMachineLearningThesis/src/datasets.jl:13
generate_data at /DistributedMachineLearningThesis/src/datasets.jl:77
generate_node_data at /DistributedMachineLearningThesis/src/workers.jl:23
#106 at ./distributed/process_messages.jl:268 [inlined]
run_work_thunk at ./distributed/process_messages.jl:56
macro expansion at ./distributed/process_messages.jl:268 [inlined]
#105 at ./event.jl:73
#remotecall_fetch#141(::Array{Any,1}, ::Function, ::Function, ::Base.Distributed.Worker, ::Function, ::Vararg{Any,N} where N) at ./distributed/remotecall.jl:359
remotecall_fetch(::Function, ::Base.Distributed.Worker, ::Function, ::Vararg{Any,N} where N) at ./distributed/remotecall.jl:351
#remotecall_fetch#144(::Array{Any,1}, ::Function, ::Function, ::Int64, ::Function, ::Vararg{Any,N} where N) at ./distributed/remotecall.jl:372
remotecall_fetch(::Function, ::Int64, ::Function, ::Vararg{Any,N} where N) at ./distributed/remotecall.jl:372
macro expansion at /DistributedMachineLearningThesis/src/master_summary.jl:186 [inlined]
(::##184#198{String,Int64,Int64,Array{Any,1},Array{Any,1}})() at ./task.jl:335

...and 3 more exception(s).

Stacktrace:
 [1] sync_end() at ./task.jl:287
 [2] macro expansion at ./task.jl:303 [inlined]
 [3] run(::Int64, ::Int64, ::String, ::Int64, ::Int64) at /DistributedMachineLearningThesis/src/master_summary.jl:183
 [4] execute_experiment() at /DistributedMachineLearningThesis/src/master_summary.jl:393
```

**Comment by jgpavez** [comment]
Might this error be produced due to different versions of julia? (code tested in julia 0.6.0)

**Comment by lagefreitas** [comment]
We'll try versions 0.6.0 and 0.6.2. Thanks!

@NaelsonDouglas please change Dockerhub to 0.6.0 and 0.6.2 (two tags) so I'll try both version.

**Comment by NaelsonDouglas** [comment]
I only tested it in 0.6.2 and never got this error.

**Comment by NaelsonDouglas** [comment]
I fixed it to use Julia 0.6.2 instead of 0.6.4 on the Dockerfile [here](https://github.com/NaelsonDouglas/DistributedMachineLearningThesis/commit/68d7cb5b4c093f6856a2aaeb099343dca07fd5d4)

**Comment by lagefreitas** [comment]
This solved this issue:

```julia
Pkg.pin("Distributions", v"0.15.0")
```

Thanks @jgpavez !
